rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -49,6 +49,11 @@ void fmt_class_string<CellSaveDataError>::format(std::string& out, u64 arg)
 	});
 }
 
+struct save_data_info
```

Comment:
Instead of having 2 FXO structs in the same cpp file, combine savedata_mutex struct to this. It's just moving one variable called `mutex` to here.

Reply:
I won't ask to revert changes (it doesn't matter much) but the original goal of things such as fxo was to split as many singletons as possible, as opposed to merging everything into a fixed mega-struct...